From 04386ca5df5aa294fa40cc526dffd0ca3c658303 Mon Sep 17 00:00:00 2001
From: Tom Stellard <tstellar@redhat.com>
Date: Thu, 12 Oct 2017 21:31:50 -0700
Subject: [PATCH] Fix -Werror=format-overflow warning

---
 src/core/runtime/hsa.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/core/runtime/hsa.cpp b/src/core/runtime/hsa.cpp
index 64129ac..3b45afd 100644
--- a/src/core/runtime/hsa.cpp
+++ b/src/core/runtime/hsa.cpp
@@ -336,7 +336,7 @@ static size_t get_extension_table_length(uint16_t extension, uint16_t major, uin
       return 0;
   }
 
-  char buff[3];
+  char buff[6];
   sprintf(buff, "%02u", minor);
   name += std::to_string(major) + "_" + buff + "_pfn_t";
 
-- 
2.13.6

